Assets/Scripts/Models/WorldGenerator.cs

```diff
@@ -80,12 +80,12 @@ public static void Generate(World world, int seed)
 
         for (int z = 0; z < depth; z++)
         {
-            float zScale = Mathf.Lerp(1f, .5f, Mathf.Abs((depth / 2f) - z) / depth);
```

Comment:
There is a StyleCop error that shows up, because it things that z is a prefix. The more correct solution would be to add z to the list of allowed prefixes, or just disable that check entirely.